fuse/utils/misc/misc.py

```diff
@@ -158,22 +158,22 @@ def time_display(seconds: int, granularity: int = 3) -> str:
 
 def get_pretty_dataframe(df: pd.DataFrame, col_width: int = 25) -> str:
     # check is col_width needs to be widen (so that dashes are in one line)
-    max_val_width = np.vectorize(len)(
-        df.values.astype(str)
-    ).max()  # get maximum length of all values
-    max_col_width = max(
-        [len(x) for x in df.columns]
-    )  # get the maximum lengths of all column names
-    col_width = max(max_col_width, max_val_width, col_width)
-
-    dashes = (col_width + 2) * len(df.columns.values)
+    max_val_width_per_col = np.vectorize(len)(df.values.astype(str)).max(
```

Comment:
Here you've modified the code to set the width per column instead of a single width for all columns?

Reply:
yes, to make the overall width smaller - so it better fits the screen